microcosm_postgres/models.py

```diff
@@ -80,6 +80,10 @@ def created_timestamp(self):
     def updated_timestamp(self):
         return (self.updated_at.replace(tzinfo=None) - EPOCH).total_seconds()
 
+    @classmethod
```

Comment:
I don't think we usually go through the model for this sort of logic. You could put this logic entirely in the store with `self.model_class.create_at.asc()` in the `order_by` func you already overwrote.

Reply:
@cdm-ium Agreed, felt a little funky.  I was worried putting that code in the store is too tightly couple the assumption that `EntityMixin` is being used as the backing model for the store since not all classes will have `create_at`... WDYT?

Reply:
You can always do something like `getattr(entity, 'sort_key')` on your model class, and no-op if not defined.